litmus-portal/frontend/src/components/Targets/InstallChaos/styles.ts

```diff
@@ -0,0 +1,158 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: '95%',
+    fontSize: 18,
+    color: theme.palette.text.secondary,
+    marginLeft: theme.spacing(8),
+    marginTop: theme.spacing(4),
+  },
+
+  linkBox: {
+    backgroundColor: ' rgba(0, 0, 0, 0.02)',
+    border: '1px solid  rgba(0, 0, 0, 0.05)',
+    padding: 30,
+    borderRadius: 8,
+    display: 'flex',
+    flexDirection: 'row',
+    wordWrap: 'break-word',
+    justifyContent: 'space-between',
+    [theme.breakpoints.down('sm')]: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+  },
+
+  buttonBox: {
+    padding: 10,
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    [theme.breakpoints.down('sm')]: {
+      display: 'flex',
+      flexDirection: 'row',
+    },
+  },
+
+  copyBtn: {
+    width: 95,
+    height: 50,
+    borderColor: theme.palette.text.primary,
+    textTransform: 'none',
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 15,
+    },
+  },
+  displayYamlBtn: {
+    width: 95,
+    height: 50,
+    borderColor: theme.palette.text.primary,
+    marginLeft: 20,
+    textTransform: 'none',
+    [theme.breakpoints.down('sm')]: {
```

Comment:
Here style for copyBtn and displayYamlBtn are same except one is having marginLeft, please avoid repetition and use `${} ${}`